<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCAN_DecodeParametersDialog</class>
 <widget class="QDialog" name="NewCAN_DecodeParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new CAN Decode Parameters</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblID">
       <property name="text">
        <string>ID [HEX]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtID">
       <property name="text">
        <string>0x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblUnit">
       <property name="text">
        <string>Unit:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtUnit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblMultiplier">
       <property name="text">
        <string>Multiplier:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtMultiplier"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblOffset">
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtOffset"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblValueMask">
       <property name="text">
        <string>Value Mask:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="chkByte0">
         <property name="text">
          <string>Byte 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkByte1">
         <property name="text">
          <string>Byte 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkByte2">
         <property name="text">
          <string>Byte 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkByte3">
         <property name="text">
          <string>Byte 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkByte4">
         <property name="text">
          <string>Byte 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkByte5">
         <property name="text">
          <string>Byte 5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkByte6">
         <property name="text">
          <string>Byte 6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkByte7">
         <property name="text">
          <string>Byte 7</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblLeftByte">
         <property name="text">
          <string>LSB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblRightByte">
         <property name="text">
          <string>MSB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblEndianness">
       <property name="text">
        <string>Endianness:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="rbLittleEndian">
         <property name="text">
          <string>Little Endian</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbBigEndian">
         <property name="text">
          <string>Big Endian</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>formLayoutWidget</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewCAN_DecodeParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewCAN_DecodeParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
